chat-client-ui-types/src/uiContracts.ts

```diff
@@ -120,6 +128,24 @@ export interface CopyCodeToClipboardMessage {
     params: CopyCodeToClipboardParams
 }
 
+export interface ShowExportChatDialogParams {}
+
+export interface ShowExportChatDialogMessage {
+    command: typeof SHOW_EXPORT_CHAT_DIALOG
+    params: ShowExportChatDialogParams
+}
+
+export interface SaveChatParams {
+    tabId: string
+    filePath: string
+    format: 'markdown' | 'html'
+}
+
+export interface SaveChatMessage {
+    command: typeof SAVE_CHAT_CONFIRMATION
```

Comment:
Is it kind of response to `SHOW_EXPORT_CHAT_DIALOG`? How about calling it just `EXPORT_CHAT`?

Where will you get `tabId` from? Should it be in `ShowExportChatDialogParams`?